panel/lxqtpanel.h

```diff
@@ -614,11 +638,27 @@ private slots:
      */
     QTimer mHideTimer;
     /**
+     * @brief QTimer for showing the panel. When the cursor is over the hidden
+     * panel area, this timer will be started. If the cursor leaves the hidden
+     * panel area, this timer will be stopped and reset. After this timer has
+     * timed out, the panel will actually be shown.
+     *
+     * \sa mHidable, mHidden, showPanel(), showPanelWork(), doShowPanel()
+     */
+    QTimer mShowTimer;
+    /**
      * @brief Stores the duration of auto-hide animation.
      *
      * \sa mHidden, mHideTimer, showPanel(), hidePanel(), hidePanelWork()
      */
     int mAnimationTime;
+    /**
+     * @brief Stores the duration of waiting time before actually restoring the
+     * panel.
+     *
+     * \sa mHidden, showPanel(), showPanelWork(), doShowPanel()
+     */
+    int mShowDelay;
```

Comment:
IMO we can get rid of this. The delay is stored in the newly added timer -> `mShowTimer.interval()`, so no need to add another member for storing the information.